addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -274,7 +274,7 @@ def _compute_from_holiday_type(self):
                 allocation.employee_ids = False
                 allocation.mode_company_id = False
             else:
-                allocation.employee_ids = self.env.context.get('default_employee_id') or self.env.user.employee_id
+                allocation.employee_ids = self.env['hr.employee'].browse(self.env.context.get('default_employee_id')) or self.env.user.employee_id
```

Comment:
Take this out of the loop, its result will not change inbetween iteration :smile: 
```py
self.env['hr.employee'].browse(self.env.context.get('default_employee_id')) or self.env.user.employee_id
```

Reply:
That's right, thanks!